### DISCUSSION TITLE
Code blocks get incorrect initial colors in dark mode

### DISCUSSION THREAD

**Discussion by weareoutman** [facebook/docusaurus#8983] [General]
Even though the colors will get back to correct ones quickly, but in poor network, people will notice that.

You can reproduce it by opening [this page](https://docusaurus.io/docs/markdown-features/react) while cache disabled and throttling sets to Fast 3G.

The code blocks will first show as white background, then after a while, get back to black background.

Here is a screenshot:

![image](https://github.com/facebook/docusaurus/assets/2338946/0e05c6ae-486e-401a-9244-6a616fbf4c51)

I consider this as not a bug, but it will be great if we can improve this. So I opened a discussion instead of an issue, thank you.

**Comment by slorber** [comment]
Yes this is a known issue due to the highlighter not using css vars but requiring React hydration.
I keep this in mind to fix it someday.
